ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -87,7 +87,7 @@ gcp)
     REGION="$(oc get -o jsonpath='{.status.platformStatus.gcp.region}' infrastructure cluster)"
     export TEST_PROVIDER="{\"type\":\"gce\",\"region\":\"${REGION}\",\"multizone\": true,\"multimaster\":true,\"projectid\":\"${PROJECT}\"}"
     ;;
-aws)
+aws | hypershift)
```

Comment:
@smarterclayton is this the right way to handle it, or should I be trying to somehow override the CLUSTER_TYPE to be aws from the outside?

Reply:
Ugh.  If the workflow were the one setting this, that would be better (to set to AWS).  Right now CLUSTER_TYPE is set at two levels (job, and then inside steps).  You may need to ask DPTP when step registry is the one that can set CLUSTER_TYPE since they were working to make some changes to how profiles and cluster type worked together.